docs/src/main/sphinx/connector/hive.md

```diff
@@ -628,14 +628,39 @@ Hive directly, most operations can be performed using Trino.
 
 #### Schema evolution
 
-Hive allows the partitions in a table to have a different schema than the
-table. This occurs when the column types of a table are changed after
-partitions already exist (that use the original column types). The Hive
-connector supports this by allowing the same conversions as Hive:
-
-- `VARCHAR` to and from `TINYINT`, `SMALLINT`, `INTEGER` and `BIGINT`
-- `REAL` to `DOUBLE`
-- Widening conversions for integers, such as `TINYINT` to `SMALLINT`
+Hive allows the partitions in a table to have a different schema than the table.
+This occurs when the column types of a table are changed after partitions
+already exist (that use the original column types). The Hive connector supports
```

Comment:
The Hive connector supports this schema evolution by allowing the same conversions as Hive.